test/utils/noderesourcetopologies/noderesourcetopologies.go

```diff
@@ -232,7 +232,7 @@ func FilterAnyZoneMatchingResources(nrts []nrtv1alpha1.NodeResourceTopology, req
 	for _, nrt := range nrts {
 		matches := 0
 		for _, zone := range nrt.Zones {
-			klog.Infof(" ----> node %q zone %q provides %s requrst %s", nrt.Name, zone.Name, e2ereslist.ToString(AvailableFromZone(zone)), reqStr)
+			klog.Infof(" ----> node %q zone %q provides %s request %s", nrt.Name, zone.Name, e2ereslist.ToString(AvailableFromZone(zone)), reqStr)
```

Comment:
arrows is a new format for us to observe things more easily ?   maybe this should be consistent, just a thought, not for this PR.

Reply:
it was mostly me trying to give a structure to the output, and this somehow stuck, probably because we lack better alternatives.